kernel: bump 6.6 to 6.6.114

zxlhhyccc opened the pull request:
kernel: bump 6.6 to 6.6.111
https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.111

All patches automatically rebased.

Build system: bcm4908
Build system: bcm53xx

kernel: bump 6.6 to 6.6.112

https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.112

Added backport
target/linux/generic/backport-6.6/542-v6.18-ksmbd-fix-recursive-locking-in-RPC-handle-list-access.patch
cherry picked from https://github.com/torvalds/linux/commit/88f170814fea74911ceab798a43cbd7c5599bed4

All patches automatically rebased.

Build system: bcm4908
Build system: bcm53xx

kernel: bump 6.6 to 6.6.113

https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.113

Manually rebased:

/target/linux/generic/backport-6.6/540-v6.12-ksmbd-browse-interfaces-list-on-FSCTL_QUERY_INTERFACE_INFO.patch

Dropped patch

/target/linux/generic/backport-6.6/541-v6.18-ksmbd-add-max-ip-connections-parameter.patch

merged upstream with commit: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=linux-6.6.y&id=bc718d0bd87e372f7786c0239e340f3577ac94fa

All other patches automatically rebased.

Build system: bcm4908
Build system: bcm53xx

kernel: bump 6.6 to 6.6.114

https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.114

Removed upstreamed:
/target/linux/generic/backport-6.6/540-v6.12-ksmbd-browse-interfaces-list-on-FSCTL_QUERY_INTERFACE_INFO.patch

merged upstream with commit: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.6.114&id=499089376206f3f7f09d6852b3ec92f2674d24bb

All other patches automatically rebased.

Build system: bcm4908
Build system: bcm53xx

zxlhhyccc committed:
kernel: bump 6.6 to 6.6.111

https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.111

All patches automatically rebased.

Build system: bcm4908
Build system: bcm53xx

Signed-off-by: Zxl hhyccc <zxlhhy@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/20434
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

zxlhhyccc committed:
kernel: bump 6.6 to 6.6.112

https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.112

All patches automatically rebased.

Build system: bcm4908
Build system: bcm53xx

Signed-off-by: Zxl hhyccc <zxlhhy@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/20434
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

PalebloodSky commented:
Thanks. Btw, 6.6.112 is out too.

hauke commented:
Please add this commit into the backports folder: https://github.com/torvalds/linux/commit/88f170814fea74911ceab798a43cbd7c5599bed4

zxlhhyccc committed:
kernel: bump 6.6 to 6.6.113

https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.113

Manually rebased:

/target/linux/generic/backport-6.6/540-v6.12-ksmbd-browse-interfaces-list-on-FSCTL_QUERY_INTERFACE_INFO.patch

Dropped patch

/target/linux/generic/backport-6.6/541-v6.18-ksmbd-add-max-ip-connections-parameter.patch

merged upstream with commit:  https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=linux-6.6.y&id=bc718d0bd87e372f7786c0239e340f3577ac94fa

All other patches automatically rebased.

Build system: bcm4908
Build system: bcm53xx

Signed-off-by: Zxl hhyccc <zxlhhy@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/20434
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

hauke commented on target/linux/generic/backport-6.6/540-v6.12-ksmbd-browse-interfaces-list-on-FSCTL_QUERY_INTERFACE_INFO.patch:
Please keep this patch.
This patch was not backported and only taking the code from upstream kernel will break the ksmb ABI. 
This will be fixed in 6.6.114.

GoetzGoerisch replied:
@zxlhhyccc 
`540-v6.12-ksmbd-browse-interfaces-list-on-FSCTL_QUERY_INTERFACE_INFO.patch` is only integrated in `.114`
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.6.114&id=499089376206f3f7f09d6852b3ec92f2674d24bb

Thus needs to be stay in `.113` bump.


`541-v6.18-ksmbd-add-max-ip-connections-parameter.patch`
is in `.113`
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.6.113&id=bc718d0bd87e372f7786c0239e340f3577ac94fa

zxlhhyccc committed:
kernel: bump 6.6 to 6.6.114

https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.114

Removed upstreamed:
/target/linux/generic/backport-6.6/540-v6.12-ksmbd-browse-interfaces-list-on-FSCTL_QUERY_INTERFACE_INFO.patch

merged upstream with commit: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.6.114&id=499089376206f3f7f09d6852b3ec92f2674d24bb

All other patches automatically rebased.

Build system: bcm4908
Build system: bcm53xx

Signed-off-by: Zxl hhyccc <zxlhhy@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/20434
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
